tests/functional/disaster-recovery/sc_arbiter/test_graceful_restart.py

```diff
@@ -75,10 +75,14 @@ def test_graceful_restart(
 
         # setup vm and write some data to the VM instance
         vm_obj = cnv_workload(volume_interface=constants.VM_VOLUME_PVC)
+        vm_obj.run_ssh_cmd(command="mkdir /test && sudo chmod -R 777 /test")
         vm_obj.run_ssh_cmd(
-            command="dd if=/dev/zero of=/file_1.txt bs=1024 count=102400"
+            command="< /dev/urandom tr -dc 'A-Za-z0-9' | head -c 10485760 > /test/file_1.txt && sync"
+        )
+        md5sum_before = cal_md5sum_vm(vm_obj, file_path="/test/file_1.txt")
+        log.debug(
+            f"This is the file_1.txt content:\n{vm_obj.run_ssh_cmd(command='cat /test/file_1.txt')}"
```

Comment:
Do we really need to change the file path?
AFAIK, md5sum is calculated based on the content of the data and does not consider any metadata about the files, such as its size, location, etc. so, I don't think the file path change is needed here. Please check